AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java

```diff
@@ -135,6 +137,13 @@ public void afterTextChanged(final Editable editable) {
                 setJoinButtonState();
             }
         });
+        setupName.setOnFocusChangeListener((view, hasFocus) -> {
```

Comment:
Could this be done with a selector?

input_text_selector.xml
```
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:state_focused="true" android:color="@color/colorAccent" />
    <item android:color="@color/textbox_secondary" /> <!-- default case -->
</selector>
```

TextView property:
```
<TextView
    android:textColor="@‌​drawable/input_text_selector" />
```

Reply:
I tried this approach before. It does not work as we do not set any focus on TextView.

Reply:
I misunderstood the change. I updated the PR title and description to better match the changes.